GitExtensions.ruleset

```diff
@@ -0,0 +1,177 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Git Extensions Rules" Description="Hand selected set of analyzer rules for use within the GitExtensions project." ToolsVersion="15.0">
+  <Include Path="minimumrecommendedrules.ruleset" Action="Default" />
+  <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
+    <Rule Id="CA1301" Action="None" />
+    <Rule Id="CA2101" Action="None" />
```

Comment:
These rules were carried over from the pre-existing FxCop stuff. Not sure if they have any effect any more.

Is FxCop still used against GE?